app/build.gradle

```diff
@@ -43,6 +43,8 @@ dependencies {
     implementation Dependencies.kotlin_stdlib
     implementation Dependencies.design
     implementation Dependencies.constraint_layout
+    implementation 'androidx.appcompat:appcompat:1.0.0-beta01'
```

Comment:
we have the dependencies on another module of the project, can you put these 2 new dependencies there?